algoritm/recursive/factorial.js

```diff
@@ -0,0 +1,17 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on("line", (N) => {
+  N = parseInt(N);
+  console.log(factorial(N));
+  rl.close();
+}).on("close", () => {
+  process.exit();
+});
+
+function factorial(N) {
+  return N === 0 ? 1 : N <= 1 ? N : N * factorial(N - 1);
```

Comment:
else if로 "?" 한번 더 작성하지 않고 아래와 같이 생략 가능해요!
```suggestion
  return N === 0 ? 1 : N *= factorial(N - 1);
```

Reply:
오호... 훨씬 간단하군... 감사합니당